cos.c

```diff
@@ -0,0 +1,12 @@
+float coseno(float z){
+  //Declaracion de variables:
+  int n;
+  float sum, termino;
+
+  //Desarrollo el algoritmo:
+  for(n=1, sum=0, termino=1 ; n<30; n++){
```

Comment:
la inicializacion de variables no deben ir, todas, dentro del for, solo la que quieres que itere, quitalas